File: 백준/Bronze/2581. 소수/소수.py
m = int(input())
n = int(input())
sosu_list = list()
number = m

while(number <= n):
    for i in range(2, number+1):
        if i == number:
            sosu_list.append(number)
        if number % i == 0:
            break
    number += 1
if len(sosu_list) == 0:
    print(-1)
elif len(sosu_list) != 0:
    print(sum(sosu_list))
    print(min(sosu_list))

File: 백준/Bronze/2581. 소수/소수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int m = input.nextInt();
        int n = input.nextInt();

        int sum = 0;
        int min = n;

        x : for (int i = m; i <= n; i++) {
            if (i == 1) {
                continue;
            }
            for (int j = 2; j < i; j++) {
                if (i % j == 0) {
                    continue x;
                }
            }
            sum += i;
            if (min > i) {
                min = i;
            }
        }
        if (sum == 0 ) {
            System.out.println(-1);
            return;
        }
        System.out.println(sum + "\n" + min);
    }
}
